### DISCUSSION TITLE
Having issues getting installed.

### DISCUSSION THREAD

**Discussion by jacen2110** [python-poetry/poetry#8356] [Q&A]
Fresh new Ubuntu 22.04.3 LTS and I ran the following commands in my terminal:

sudo apt update
sudo apt upgrade
sudo apt install python3
sudo apt install python3-pip
pip3 install poetry
poetry install

and I get this error:

Command 'poetry' not found, but can be installed with:
sudo apt install python3-poetry

**Comment by dimbleby** [comment]
https://python-poetry.org/docs/#installation

**Reply by jacen2110** [reply]
I should've mentioned I already have been through the installation page.

**Reply by dimbleby** [reply]
but you haven't done any of the things that it says, and you are instead doing something else?

please follow the docs

**Reply by jacen2110** [reply]
I just said that I did all of that already ....(?)
